sc3-plugins 3.14.1 adds -stdlib=libc++ under clang. The SSP buildroot has only libstdc++ (gcc 8.4).

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -120,9 +120,6 @@
 	endif()
 
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++17")
-    if(CMAKE_COMPILER_IS_CLANG)
-      set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -stdlib=libc++")
-    endif()
 endif()
 if(MINGW)
 	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mstackrealign")
